src/gluonts/dataset/loader.py

```diff
@@ -89,42 +94,39 @@ def __init__(
             assert max_queue_size >= num_workers
 
         self.decode_fn = decode_fn
-        self.result_queue = mp.Manager().Queue(maxsize=max_queue_size)
-
-        create_worker = partial(
-            mp.Process,
-            target=worker_fn,
-            kwargs={
-                "dataset": dataset,
-                "result_queue": self.result_queue,
-                "num_workers": num_workers,
-            },
-        )
+        self.queue_timeout_seconds = queue_timeout_seconds
+        self.output_queue = mp.Manager().Queue(maxsize=max_queue_size)
```

Comment:
Does it matter if we create multiple managers?

Would something change if we just use one?

Reply:
Good spot, I'll update this. We were using just one queue so far, that's why it was created on the spot